mapeditor/inspector/inspector.cpp

```diff
@@ -464,7 +464,7 @@ void Inspector::updateProperties()
 		return;
 	table->setRowCount(0); //cleanup table
 	
-	addProperty("Indentifier", obj);
+	addProperty("Identifier", obj);
```

Comment:
Would this be problematic?

Reply:
This should be safe. Seems to be only used inside of map editor